src/styles/blocks/price.scss

```diff
@@ -0,0 +1,23 @@
+.card__price {
+  margin: 0 16px 16px;
```

Comment:
Remove the horizontall margins everywhere and add instead the padding for the card